.circleci/config.yml

```diff
@@ -32,23 +32,22 @@ parameters:
     type: string
     default: ""
 
-aliases:
-  base-job: &base-job
-    resource_class: macos.m1.medium.gen1
-    macos:
-      xcode: << parameters.xcode_version >>
+executors:
+  macos-executor:
     parameters:
       xcode_version:
         type: string
         default: '15.3'
-      install_swiftlint:
-        type: boolean
-        default: true
```

Comment:
This was honestly a pretty deceiving one - you would have to re-define it as a param, but then also remember to pass it in, which was kinda weird (you'll see what I mean in other comments). 
And it's not super useful, so I removed it so we can use it at the job level instead